pkg/preparer/listener.go

```diff
@@ -124,6 +124,8 @@ func (l *HookListener) Sync(quit <-chan struct{}, errCh chan<- error) {
 				sub.WithField("err", err).Errorln("Could not write hook link")
 			} else {
 				sub.NoFields().Infoln("Updated hook")
+				// Send back a nil error to signal success!
+				errCh <- nil
```

Comment:
We shouldn't use nil as a sentinel value for success. One wrong access and this whole process panics.

There are a few alternatives I can think of - one, we actually have the preparer type provide an `eventStream` channel. The status handler code could `select` on the event stream success and the error channel. This has the benefit of being useful in other ways (for example, to create a local history).

A less preferable approach creates a `MaybeError` channel.